vnext/package.json

```diff
@@ -69,5 +69,56 @@
       "minor",
       "patch"
     ]
-  }
+  },
+  "files": [
+    "/codegen",
+    "/Chakra",
+    "/Common",
+    "/DeforkingPatches",
+    "/etc",
+    "/flow",
+    "/Folly",
+    "/include",
+    "/jest",
+    "/JSI",
+    "/Libraries",
+    "!/Libraries/**/*.windesktop.*",
+    "/local-cli",
+    "/Microsoft.ReactNative",
+    "/Microsoft.ReactNative.Cxx",
+    "/Microsoft.ReactNative.Managed",
+    "/Microsoft.ReactNative.SharedManaged",
+    "/Mso",
+    "/PropertySheets",
+    "/ReactCommon",
+    "/ReactUWP",
+    "/ReactWindowsCore",
+    "/Scripts",
+    "/Shared",
+    "/stubs",
+    "/Test",
```

Comment:
What's in `Test`?

Reply:
This was http server and web sockets. They were only used by test projects. Removed...